File: solutions/72-rafidhaque/src/BangladeshiFarmers.java
import javafx.scene.Group;
import javafx.scene.paint.Color;
import javafx.scene.shape.Rectangle;

public class BangladeshiFarmers implements INation {

    String nationName = "BangladeshiFarmers";
    Group drawSpace;
    DrawHouse house;
    DrawTree tree;
    DrawWater water;

    public BangladeshiFarmers(Group drawSpace) {
        this.drawSpace = drawSpace;
        Rectangle fillColor = new Rectangle(0,0, 600, 400);
        fillColor.setFill(Color.GREENYELLOW);
        drawSpace.getChildren().add(fillColor);
        house = new BangladeshiFarmersHouse();
        tree = new BangladeshiFarmersTree();
        water = new BangladeshiFarmersWater(24, 16);
        drawSpace.getChildren().addAll(house.draw());
        drawSpace.getChildren().addAll(tree.draw());
        drawSpace.getChildren().addAll(water.draw());
    }

    public String getNationName() {
        return nationName;
    }

    public DrawHouse getHouse() {
        return house;
    }

    public DrawTree getTree() {
        return tree;
    }

    public DrawWater getWater() {
        return water;
    }
}

File: solutions/73-tasnim/AgeOfVillagers/AgeOfVillagers/BangladeshiFarmers.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AgeOfVillagers
{
    class BangladeshiFarmers : INation
    {
        public void Draw_House(Graphics g, Point p)
        {
            BangladeshiFarmersHouse house = new BangladeshiFarmersHouse(p);
            house.Draw_Shape(g);
        }

        public void Draw_Tree(Graphics g, Point p)
        {
            BangladeshiFarmersTree Tree = new BangladeshiFarmersTree(p);
            Tree.Draw_Shape(g);
        }

        public void Draw_WaterSource(Graphics g, Point p)
        {
            BangladeshiFarmersWater water = new BangladeshiFarmersWater(p);
            water.Draw_Shape(g);
        }

        public Color Color_Terrain()
        {
            return Color.Green;
        }
    }
}
